packages/components/src/presentation-components/tree/controlled/PresentationTreeRenderer.tsx

```diff
@@ -116,13 +117,22 @@ function MatchingInstancesCount({ filter, dataProvider, parentKey }: MatchingIns
   const { value, inProgress } = useDebouncedAsyncValue(
     useCallback(async () => {
       const instanceFilter = await convertToInstanceFilterDefinition(filter.filter, dataProvider.imodel);
+      const requestOptions = dataProvider.createRequestOptions(parentKey, instanceFilter);
 
       try {
-        const count = await Presentation.presentation.getNodesCount(dataProvider.createRequestOptions(parentKey, instanceFilter));
-        return `${translate("tree.filter-dialog.results-count")}: ${count}`;
+        const count = await Presentation.presentation.getNodesCount(requestOptions);
+        return <>{`${translate("tree.filter-dialog.results-count")}: ${count}`}</>;
```

Comment:
Why different styling for `tree.filter-dialog.results-count` message and result set too large message?

Reply:
Because I wanted the informational messages to match the ones that are displayed in the tree. Also, I thought that it is a good idea to have different styling for messages when everything goes well and another styling for _when something goes wrong_.

Reply:
I don't think in this case `ResultSetTooLarge` error means that something went wrong. We just want to provide some indication to user if current filter is enough to limit hierarchy level to required size or not. So both messages should have same format in my opinion.

Reply:
Okay, I will change it to match

Reply:
I guess it could be changed back to return `string` instead of `React.Fragment`